src/main/resources/checkstyletask.properties

```diff
@@ -1 +0,0 @@
-checkstyle=com.puppycrawl.tools.checkstyle.CheckStyleTask
```

Comment:
hmm, very strange, try one more time, if still a problem I will apply PR as is (unfortunate , but not blocking issue)